community/sway/etc/sway/definitions

```diff
@@ -24,9 +24,11 @@ set $background /usr/share/backgrounds/manjaro-sway/gruvbox-dark-manjarosway-384
 set $theme /usr/share/sway/themes/matcha-green
 
 # Your preferred terminal emulator
-set $term /usr/share/sway/scripts/terminal.sh
-set $term_float $term float
-set $term_float_portrait $term float_portrait
+set $term foot
+set $term_client footclient
```

Comment:
```suggestion
# make a foot client instance the default terminal for even faster terminal windows
# set $term footclient
```

Reply:
It's for launching floating windows with $term_client. Though my current setup is to statically use 'footclient -a floating_shell', I just wasn't sure if it's good enough to leave it static, so I made this solution, which I don't feel is great either.

Reply:
It would be wise to move the suggestion that you made to the etc/skel/.config/sway/config.d/00-user.conf for example, so it will be easier, just uncomment it and you're done. Probably adding a word for low-memory footprint or\and potential data loss if foot --server crashes would be nice, too

Reply:
So just to clarify: if we're getting rid of $term_client, are we going to replace $term_float to static `footclient` command, or we're using $term here to simplify configuration of third-party terminal? Because then these optimizations for floating windows are lost, unfortunately

Reply:
You're absolutely right. Let's just keep it like it is now and add a nice hint to the 00-user.conf